parachain/pallets/ethereum-beacon-client/fuzz/README.md

```diff
@@ -0,0 +1,21 @@
+# Beacon client fuzz tests
+
+This crate contains fuzz tests for the three beacon client extrinsics.
+
+# Installation
+
+```
+cargo install cargo-fuzz
+```
+
+# Run tests
+
+- Force Checkpoint: `cargo fuzz run fuzz_force_checkpoint -- -max_len=10000000000`
+- Submit: `cargo fuzz run fuzz_submit -- -max_len=10000000000`
+- Submit Execution Header: `cargo fuzz run fuzz_submit_execution_header -- -max_len=10000000000`
+
+Note: `max-len` is necessary because the max input length is 4096 bytes. Some of our inputs are larger than this
+default value. When running the tests without an increased max len parameter, no fuzz data will be generated.
```

Comment:
Nice!

Reply:
Added in https://github.com/Snowfork/snowbridge/pull/907/commits/2ca536e7c5e0ae7b43fe1134bf297410690cd8b9.